docs/docs/installation.mdx

```diff
@@ -173,11 +172,7 @@ without root privileges.
 Next step: Start prototyping your first assistant online and download it afterwards
 
 
-<a className="button button--outline button--secondary button--lg" href="prototype-an-assistant">Prototype an Assistant</a>
-
----
-
-<a aria-hidden="true" tabIndex="-1" className="anchor enhancedAnchor" id="build-from-source"></a>
+<Button as="a" href="prototype-an-assistant">Prototype an Assistant</Button>
```

Comment:
Is the red effect intended when hovering the button?

<img width="276" alt="Screenshot 2020-09-08 at 14 36 32" src="https://user-images.githubusercontent.com/1931615/92477329-b3d0df00-f1e0-11ea-90a4-7766c40d9da4.png">

Reply:
This is a placeholder for now, the styling isn't complete.